### PR TITLE
CC-25924: Update Customer Account Management feature integration documentation.

### PR DESCRIPTION
**Pull request by michbeck**
## PR Description

Ticket: https://spryker.atlassian.net/browse/CC-25924
PR: https://github.com/spryker/suite-nonsplit/pull/7114

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: _includes/pbc/all/install-features/202304.0/install-the-customer-account-management-feature.md [hunk 1]
```diff
@@ -0,0 +1,1198 @@
...
+Ensure that the following modules have been installed:
+
+| MODULE                  | EXPECTED DIRECTORY                           |
+|-------------------------|----------------------------------------------|
+| Customer                | vendor/spryker/customer                      |
+| CustomerGroup           | vendor/spryker/customer-group                |
+| CustomerNote            | vendor/spryker/customer-note                 |
+| CustomerNoteGui         | vendor/spryker/customer-note-gui             |
+| CustomerStorage         | vendor/spryker/customer-storage              |
+| CustomerValidationPage  | vendor/spryker-shop/customer-validation-page |
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202304.0/install-the-customer-account-management-feature.md:48]: this module and related to it things should be moved to respective section about frontend

File: _includes/pbc/all/install-features/202304.0/install-the-customer-account-management-feature.md [hunk 2]
```diff
@@ -0,0 +1,1198 @@
...
+
+```php
+$config[OauthConstants::PRIVATE_KEY_PATH] = 'file://path/to/private.key';
+$config[OauthConstants::PUBLIC_KEY_PATH] = 'file://path/to/public.key';
+$config[OauthConstants::ENCRYPTION_KEY] = 'generated-encryption-key';
+$config[OauthConstants::OAUTH_CLIENT_IDENTIFIER] = 'client-identifier';
+$config[OauthConstants::OAUTH_CLIENT_SECRET] = 'client-secret';
+```
+
+2. Register publish and synchronization queue:
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202304.0/install-the-customer-account-management-feature.md:77]: it's more like `Add queue configuration` or something like this
please check similar guides referring this file

File: _includes/pbc/all/install-features/202304.0/install-the-customer-account-management-feature.md [hunk 3]
```diff
@@ -0,0 +1,1198 @@
...
+
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202304.0/install-the-customer-account-management-feature.md:78]: I see `src/Pyz/Zed/Customer/Persistence/Propel/Schema/spy_customer.schema.xml` is adjusted at nonsplit PR but not mentioned

**Review by DmitryLymarenko — CHANGES_REQUESTED** [review]

File: _includes/pbc/all/install-features/202304.0/install-the-customer-account-management-feature.md [hunk 4]
```diff
@@ -0,0 +1,1198 @@
...
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return array<\Symfony\Component\Console\Command\Command>
+     */
+    protected function getConsoleCommands(Container $container): array
+    {
+        return [
+            new CustomerPasswordResetConsole(),
+            new CustomerPasswordSetConsole(),
+            new DeleteExpiredCustomerInvalidatedRecordsConsole(),
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202304.0/install-the-customer-account-management-feature.md:592]: this plugin is missing at table with plugin descriptions

File: _includes/pbc/all/install-features/202304.0/install-the-customer-account-management-feature.md [hunk 5]
```diff
@@ -0,0 +1,1198 @@
...
+    }
+
+    /**
+     * @return array<\SprykerShop\Yves\ShopApplicationExtension\Dependency\Plugin\FilterControllerEventHandlerPluginInterface>
+     */
+    protected function getFilterControllerEventSubscriberPlugins(): array
+    {
+        return [
+            ...
+            new LogoutInvalidatedCustomerFilterControllerEventHandlerPlugin(),
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202304.0/install-the-customer-account-management-feature.md:1050]: the same

please recheck plugins and make sure they all are mentioned at respective tables

**Review by DmitryLymarenko — DISMISSED** [review]

**Review by KarinOf — DISMISSED** [review]

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**